clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -183,6 +183,10 @@ Changes in existing checks
   by adding the option `UseUpperCaseLiteralSuffix` to select the
   case of the literal suffix in fixes.
 
+- Improved :doc:`readability-redundant-casting
+  <clang-tidy/checks/readability/redundant-casting>` check to
+  exclude `CXXParenListInitExpr` for the source expression matcher.
```

Comment:
This seems to be too technical and internal to end user. What should be observed on that side?

Reply:
Thanks, I agree it should be transparent to the user.